### DISCUSSION TITLE
Unnecessary pages being pre-fetched in development server

### DISCUSSION THREAD

**Discussion by praveshishere** [nuxt/nuxt#16379] [Questions]
When loading the homepage of my project, many of the other pages are being fetched initially, which isn't required.

So I created a dummy nuxt app with three routes
- Home `/`
- About '/about'
- Dashboard `/dashboard`

And tried the same thing here, but I find the behavior similar, all pages are shown in the network tab initially, irrespective of which route I access.
![image](https://user-images.githubusercontent.com/95841565/188872522-b6e2fd22-bd83-4da0-96a5-0d93d1fb2847.png)

Is this expected, and if so, why is it required?

**Comment by praveshishere** [comment]
I just now noticed, that only meta from the files is being imported, which is obviously required.

**Comment by tsulatsitamim** [comment]
this also happen to me, but in production, and not just meta but the full script
